### PR TITLE
Upgrade to latest shelljs

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: external/shared [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 14e5606d366e4b9711d4085b18ac0fa2e5a65ce0
+Subproject commit 03cc709564f6052f3bb0ff44c0350494f1bfab2f
```
Comment (kchitalia) [inline external/shared]: Bring in latest shared repo

File: node/forcedroid.js [hunk 2]
```diff
@@ -443,7 +443,7 @@ function fixAppGradleFileHelper(appFolderName, appName, originalDependency, newD
     console.log('Tweaking build.gradle in ' + appFolderName + "/" + appName);
     var originalDependency = "compile project(':libs:" + originalDependency + "')";
     var newDependency = "compile project(':forcedroid:libs:" + newDependency + "')";
-    shelljs.sed('-i', originalDependency, newDependency, path.join(appFolderName, appName, "build.gradle"));
+    miscUtils.replaceTextInFile(path.join(appFolderName, appName, "build.gradle"), originalDependency, newDependency);
```
Comment (kchitalia) [inline node/forcedroid.js]: Calling the replaceTextInFile function from shared repo

File: package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     ],
     "bin" : { "forcedroid" : "node/forcedroid.js" },
     "dependencies": {
-        "shelljs": "~0.5.3"
+        "shelljs": "~0.7.0"
```
Comment (kchitalia) [inline package.json]: Upgrading to latest shelljs

**Comment by bhariharan** [comment]
Looks good! 👍

**Comment by bhariharan** [comment]
I think there might be more instances of ```sed``` usage in this file though.

**Comment by kchitalia** [comment]
Can I send in a separate PR for that, I'll replace all sed commands with our new function.

**Comment by bhariharan** [comment]
Sounds good!

**Pull request merged**